tests/unittests/InterpreterTest.cpp

```diff
@@ -898,6 +898,89 @@ TEST(Interpreter, nonLinearClassifier) {
   }
 }
 
+/// Generate images in two classes. A "line" is labeled as 0
+/// and a "cross" is labeled as 1
+static void generateImageData(Tensor &images, Tensor &labels, unsigned numSamples, bool isTrain) {
+  auto L = labels.getHandle<size_t>();
+  auto image = images.getHandle<>();
+  images.zero();
+  
+  for (size_t i = 0; i < numSamples; i++) {
+    bool isLine = i % 2 == 0;
+    L.at({i, 0}) = isLine ? 0 : 1;
+    size_t target, end;
+    target = nextRandInt(2, 5);
+    // to make sure testing set is not totally included in 
+    // training set
+    end = isTrain ? 6 : 7;
+
+    for (size_t x = 0; x < end; x++) {
+      for (size_t y = 0; y < end; y++) {
+        if (isLine) {
+	  // line
+	  if (x == target)
+            image.at({i, x, y, 0u}) = 1;
+        } else {
+          // cross
+          if (x == target || y == target)
+            image.at({i, x, y, 0u}) = 1;
+        }
+      }
+    }
+  }
+}
+
+TEST(Interpreter, simpleConv) {
```

Comment:
This test needs a better name. simpleConv does not describe the test. 

Please add a comment above the test that describes what it does and what it checks.